File: Editorial/0101-0200/0136_Single_Number/CPP/1_List_operation.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        vector<int> no_duplicate_list;
        for (int i : nums) {
            if (find(no_duplicate_list.begin(), no_duplicate_list.end(), i) ==
                no_duplicate_list.end()) {
                no_duplicate_list.push_back(i);
            } else {
                no_duplicate_list.erase(remove(no_duplicate_list.begin(),
                                               no_duplicate_list.end(), i));
            }
        }
        return no_duplicate_list[0];
    }
};

File: Editorial/0101-0200/0136_Single_Number/Go/1_List_operation.go
func singleNumber(nums []int) int {
    no_duplicate_list := []int{}
    for _, i := range nums {
        found := false
        for j, x := range no_duplicate_list {
            if i == x {
                found = true
                no_duplicate_list = append(
                    no_duplicate_list[:j],
                    no_duplicate_list[j+1:]...)
                break
            }
        }
        if !found {
            no_duplicate_list = append(no_duplicate_list, i)
        }
    }
    return no_duplicate_list[0]
}
